Modules/HTMLLearningModule/classes/class.ilObjFileBasedLMGUI.php

```diff
@@ -661,7 +655,7 @@ protected function afterImport(ilObject $new_object) : void
 
     public function createFromDirectory(string $a_dir) : void
     {
-        if (!$this->checkPermissionBool("create", "", "htlm") || $a_dir == "") {
+        if ($a_dir === "" || !$this->checkPermissionBool("create", "", "htlm")) {
```

Comment:
@mjansenDatabay  This is performance related, right? And these are manual changes, or is there any automatic help for this kind of stuff?

Reply:
Yes, this is performance related. Yes, these are manual changes reported by the free `PHP EA` plugin of `PhpStorm`, which adds additional inspection results to our inspection profile. If (estimated but not needed) time is left during my review, I'll sometimes fix such trivial issues.

Reply:
Great. Thanks!